plugins/modules/collaborator_information.py

```diff
@@ -79,9 +85,28 @@
     "permissions":          url for hooks (as string),
 '''
 
-import json
-from github import Github
-from ansible.module_utils.basic import AnsibleModule
+
+def add_collaberators(repos, collaberators,):
```

Comment:
```suggestion
def add_collaborators_(repos, collaborators_):
```
if this is to avoid conflicts with keywords, it is python practice to shove a underscore onto the back